_posts/2020-07-12-changelog-126.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #126"
+date: 2020-07-13 16:54:50 +0300
+---
+
+
+In this release, we enable the new macro expansion engine by default. It was a quite long story for us,
+ so we decided to bump the version of our plugin from `0.2` to `0.3`. Meet the `IntelliJ Rust 0.3` release!
+ We'll release a dedicated blog post about the new expansion engine in [CLion blog] soon.
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5716 -->
+* Enable new macro expansion engine by default. Let's remember why the new engine is better:
+  * it supports `impl` blocks and `mod` declarations generated by declarative macros, i.e.
+   if some methods are inside `impl` block generated by macro invocation, the plugin makes name resolution
+   and completions for these methods as expected. Ability to make proper name resolution for `mod` declarations
+   provides proper name resolution and completion for crates like `tokio`, `async-std`, `reqwest`, etc.
+  * it's now possible to find macro-generated items by `Go To Struct Declaration` <kbd>Ctrl + N</kbd> or `Go To Symbol`
+   <kbd>Ctrl + Shift + N</kbd>. You can try it with standard atomic types, e.g. `AtomicI16`;
+  * it brings semantic highlighting in macro invocations;
+  * most of the IDE features are now works inside macro invocations just like in a regular code: code completion,
```

Comment:
```suggestion
  * most of the IDE features now work inside macro invocations just like in a regular code: code completion,
```

Reply:
About code completion - is it true? Looks like it's not although it works much better